consensus/reactor.go

```diff
@@ -105,7 +107,7 @@ func (conR *Reactor) OnStop() {
 // SwitchToConsensus switches from block_sync mode to consensus mode.
 // It resets the state, turns off block_sync, and starts the consensus state-machine
 func (conR *Reactor) SwitchToConsensus(state sm.State, skipWAL bool) {
-	conR.Logger.Info("SwitchToConsensus")
+	conR.Logger.Debug("Consensus SwitchToConsensus")
```

Comment:
This seems like it should still be at info level to assist with debugging (assuming that most operators will always have their logs at info level and above, and never at debug level).

Reply:
Agreed. This message should be printed only once, which for me is a clear case for Info level.